users_ldap_populate/model/users_ldap.py

```diff
@@ -20,10 +20,16 @@
 ##############################################################################
 
 import re
-from ldap.filter import filter_format
 from openerp.osv import orm
 import logging
 
+_logger = logging.getLogger(__name__)
+
+try:
+    from ldap.filter import filter_format
+except ImportError:
+    _logger.debug('Can not `from ldap.filter import filter_format`.')
```

Comment:
Being discussed in https://github.com/OCA/account-financial-tools/pull/281/files#r37519322